tasks/authentication.yml

```diff
@@ -16,12 +16,12 @@
   service:
     name: influxdb
     state: restarted
-  when: influxdb_user_password
+  when: (influxdb_user_password)
```

Comment:
I have used this form because it deals with the both empty string and null value. I have tested both cases. 

Reference: https://medium.com/opsops/wft-bool-filter-in-ansible-e7e2fd7a148f

Reply:
I am starting to dislike more and more ansible with this finding, thank you for explaining!